protos/protos/public/modeldb/ExperimentRunService.proto

```diff
@@ -564,6 +564,17 @@ message ListBlobExperimentRunsRequest {
     }
 }
 
+message CloneExperimentRun {
+    string src_experiment_run_id = 1; // Which experimentRun want to clone [Compulsory]
+    string dest_experiment_id = 2; // Destination experiment_id to clone run into. if not define here then the backend take experiment_id which is exists in src_experiment_run. [Optional]
+    string dest_project_id = 3; // Destination project_id to clone run into. if not define here then the backend take project_id which is exists in src_experiment_run. [Optional]
+    string new_owner = 4; // Owner id of new cloned experiment_run. if not define here then the backend take the login user as a owner [Optional]
```

Comment:
```suggestion
    string src_experiment_run_id = 1; // Source experiment_run to clone [Compulsory]
    string dest_experiment_run_name= 2; // Source experiment_run to clone [Compulsory]
    string dest_experiment_id = 3; // Destination experiment_id to clone run into. if not define here then value will be experiment_id of src_experiment_run_id[Optional]
    string dest_project_id = 4; // Destination project_id to clone run into. if not define here then the value will be project_id of src_experiment_run_id[Optional].
    //if both project ID and experiment ID are provided, then backend will check that the experiment is inside the project, otherwise throw a 400.
   //if no project ID and yes experiment ID is provided, then backend will fill in project ID based on the experiment
```